### ISSUE TITLE
Ctrl+Shift+Enter in print() causes internal error

### ISSUE THREAD

**Issue by ijay99** [spyder-ide/spyder#14795]
### What steps will reproduce the problem?

On Windows 10, Spyder 4.2.1
Type the code below and when still in the print function parenthesis () type message and then press Ctrl+Shift+Enter to go to a new line and the internal error will display. This keyboard shortcut is OK for other functions.

message = "Hello world!"
print(message)

### Traceback
```python-traceback
  File "C:\Users\Iain\anaconda3\lib\site-packages\spyder\plugins\editor\extensions\snippets.py", line 52, in wrapper
    return f(self, *args, **kwargs)
  File "C:\Users\Iain\anaconda3\lib\site-packages\spyder\plugins\editor\extensions\snippets.py", line 237, in _process_text
    self.insert_text(text, line, column)
  File "C:\Users\Iain\anaconda3\lib\site-packages\spyder\plugins\editor\extensions\snippets.py", line 379, in insert_text
    position = ((x, y), (x, y + 1))
TypeError: can only concatenate tuple (not "int") to tuple
```

## Versions

* Spyder version: 4.2.1 
* Python version: 3.8.5
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  7.20.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  22.0.1 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.8.0 (OK)
pygments >=2.0                  :  2.7.4 (OK)
pylint >=1.0                    :  2.6.0 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.0             :  0.3.0 (OK)
qdarkstyle >=2.8                :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.1 (OK)
qtconsole >=5.0.1               :  5.0.2 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.4 (OK)
setuptools >=39.0.0             :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                  :  3.4.3 (OK)
spyder_kernels >=1.10.1;<1.11.0 :  1.10.1 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3               :  1.0.2 (OK)
zmq >=17                        :  20.0.0 (OK)

# Optional:
cython >=0.21                   :  0.29.21 (OK)
matplotlib >=2.0.0              :  3.3.2 (OK)
numpy >=1.7                     :  1.19.2 (OK)
pandas >=1.1.1                  :  1.2.1 (OK)
scipy >=0.17.0                  :  1.6.0 (OK)
sympy >=0.7.3                   :  1.7.1 (OK)
```

**Comment by steff456** [comment]
Hi @ijay99,

I can't reproduce this issue, can you please upload a gif with the behavior you are experiencing?

**Comment by ijay99** [comment]
Hi Steff. Yes I'll do that. To reproduce it your cursor still needs to be inside the parenthesis when you type ctrl+shift+enter.

**Comment by ijay99** [comment]
Hi @steff456 

Here is the animated Gif. Note I don't move my cursor after completing the text in the parenthesis. As soon as I type ctrl+shift+enter the error appears. I've checked on another install on another machine and I get the same error.
![210224_spyder_error](https://user-images.githubusercontent.com/54008578/109065645-cc2bac00-76e3-11eb-8d42-eae6673ab1de.gif)

**Comment by ijay99** [comment]
Appears to have been fixed in release 4.2.3.
